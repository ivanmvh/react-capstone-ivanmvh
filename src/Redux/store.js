import { combineReducers, configureStore } from '@reduxjs/toolkit';
import photosSlice from './PhotosSlice';

const photosReducers = combineReducers({
  Photos: photosSlice,
});

const store = configureStore({
  reducer: photosReducers,
});

export default store;
