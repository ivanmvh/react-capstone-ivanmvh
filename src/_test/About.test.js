import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import About from '../Pages/About';
import '@testing-library/jest-dom';
import store from '../Redux/store';

describe('About', () => {
  const tree = test('renders About component', () => {
    render(
      <Provider store={store}>
        <BrowserRouter>
          <About />
        </BrowserRouter>
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
