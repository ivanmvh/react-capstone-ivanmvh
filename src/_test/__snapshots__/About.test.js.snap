// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders About component 1`] = `undefined`;

exports[`Navbar renders Navbar component 1`] = `undefined`;
